src/reducers/petitions.js

```diff
@@ -15,6 +16,10 @@ export default function petitions (state = initialState, action) {
       return Object.assign({}, state,
         action.petitions, { isLoading: false }
       );
+    case UPDATE_CITY_FILTER_VALUE:
```

Comment:
I don‘t get what you mean? This action is only there to update the input value with the chosen suggestion from the search input.
Or are you saying this for when we want to prefill the input / change the headline?